diffkemp/simpll/Utils.cpp

```diff
@@ -39,6 +39,28 @@
 #include <set>
 #include <sstream>
 
+#if LLVM_VERSION_MAJOR >= 18
+
+bool refHasSuffix(StringRef ref, StringRef suffix) {
```

Comment:
`hasSuffix` is IMHO a better name.

Reply:
That should be IMO fine since that one has just one param and C++ has function overloading.

But even better, I'd rename the other one to something like `hasNumberSuffix` to better express its purpose.